rtf

```diff
@@ -802,41 +802,47 @@ class Platform:
         rudder_url = host.get_url()
         token = host.run('cat /root/rudder-token')
     if rudder_url is None or token is None:
-      print("This platform has no rudder server, can't run this command")
-      exit(2)
+      rudder_url = ''
+      token = ''
+    #  print("This platform has no rudder server, can't run this command")
+    #  exit(2)
     return (rudder_url, token)
 
-  def run_scenario(self, name, frmt, run_finally, err_stop, run_only, client_path, params, startTestNumber):
+  def run_scenario(self, name, frmt, run_finally, err_stop, run_only, client_path, params, startTestNumber, destroyOnError=False):
     """ Run a scenario on this platform """
-    # test ruby binary
```

Comment:
Add a try/catch to destroy the vm if an error occured while running the scenario